server/main.ts

```diff
@@ -127,6 +140,7 @@ Meteor.startup(() => {
         updateBlock(number);
         updateTokenPrice();
         updatePendingTransactions();
+        getAllProposals();
```

Comment:
Does it mean it will only be called once at startup?

Reply:
Right! I have added a timer to `getAllProposals()` method to call every 5 blocks